src/TestRunnerOrchestrator.ts

```diff
@@ -168,7 +168,7 @@ export default class TestRunnerOrchestrator {
   }
 
   private createTestRunnerSandboxes(): Promise<TestRunnerSandbox[]> {
-    let cpuCount = os.cpus().length;
+    let cpuCount = this.options.cpuCount || os.cpus().length;
```

Comment:
I do prefer mentioning the test runner. That way you know what it's about. How does `maxConcurrentTestRunners` sound?